BE/api-server/src/main/resources/application.yaml

```
@@ -1,12 +1,12 @@
-spring:
-  application:
-    name: ShortLink
-
-  data:
-    redis:
-      host: ${REDIS_HOST:localhost}
-      port: ${REDIS_PORT:6379}
-      password: ${REDIS_PASSWORD:}
-
-server:
+spring:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)